job-server/src/spark.jobserver/JobManagerActor.scala

```diff
@@ -20,9 +24,11 @@ object JobManagerActor {
   case class StartJob(appName: String, classPath: String, config: Config,
                       subscribedEvents: Set[Class[_]])
   case class KillJob(jobId: String)
+  case object ContextConfig
```

Comment:
It's slightly confusing to have the return message and the sending message be the same name.  How about rename this to `GetContextConfig`?